app/views/public/_my_saved_post.html.erb

```diff
@@ -13,4 +14,7 @@
       </div>
     </div>
   </div>
-<% end %>
+ <% end %>
+<% else %>
+ <h4 class = "text-secondary nil-message" >hmm... looks like you haven't saved anything yet </h4> 
```

Comment:
please move error or success messages to locale.yml @ksowndar 
check with @vasanthakumar-a for details

Reply:
Moved @iMagesh